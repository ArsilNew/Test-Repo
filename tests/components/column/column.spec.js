import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import Column from "../../../src/components/column/column";

describe("Column", () => {
  test("default snapshot", () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<Column />);
    expect(tree).toMatchSnapshot();
  });

  test("snapshot with children", () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
      <Column>
        <div>Some kind of child</div>
      </Column>
    );
    expect(tree).toMatchSnapshot();
  });
});
