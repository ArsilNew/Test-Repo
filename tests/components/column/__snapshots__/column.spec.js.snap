// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column default snapshot 1`] = `
<div
  className="column"
/>
`;

exports[`Column snapshot with children 1`] = `
<div
  className="column"
>
  <div>
    Some kind of child
  </div>
</div>
`;
